app/host/dashboard/components/GroupActions.tsx

```diff
@@ -57,6 +56,7 @@ export default function GroupActions({ group }: { group: Tables<"groups"> }) {
 						<button
 							className="btn btn-dangerous transition-none"
 							onClick={async () => {
+								toast.success(`Deleted class "${group.name}"`);
```

Comment:
Is this toast styled properly? Also, we should implement some sort of undo so that it doesn’t actually delete until 5 seconds after